sklearn/metrics/_classification.py

```diff
@@ -1970,7 +1970,13 @@ def precision_recall_fscore_support(
         "y_pred": ["array-like", "sparse matrix"],
         "labels": ["array-like", None],
         "sample_weight": ["array-like", None],
-        "raise_warning": ["boolean"],
+        "raise_warning": ["boolean", Hidden(StrOptions({"deprecated"}))],
+        "replace_undefined_by": [
+            Hidden(StrOptions({"default"})),
+            (StrOptions({"worst"})),
```

Comment:
```suggestion
            1.0,
```

So here I used to think the worst case for the two scores is different (0 and 1), but if for both of them it's `1`, then I think it makes sense here for the argument option to be simply `1`, while explaining in the docstring (as it's already done in this PR), what the ranges for the two scores are.

cc @glemaitre

Reply:
I tend to think that `"worst"` is more intuitive than `1.0`, but it's true that this then would be different than the other cases where we have implemented `zero_division`.
But no strong opinion.